src/shared/sam/debugger/awsSamDebugger.ts

```diff
@@ -257,33 +257,13 @@ export class SamDebugConfigProvider implements vscode.DebugConfigurationProvider
         )
 
         if (!hasLaunchJson) {
-            // Try to generate a default config dynamically.
-            const configs: AwsSamDebuggerConfiguration[] | undefined = await this.provideDebugConfigurations(
-                folder,
-                token
-            )
-
-            if (!configs || configs.length === 0) {
-                getLogger().error(
-                    `SAM debug: failed to generate config (found CFN templates: ${cftRegistry.registeredTemplates.length})`
+            vscode.window.showErrorMessage(
```

Comment:
It would be cool if the message box had a button to take users to the Run panel

Reply:
done, adding screenshot